.travis.yml

```diff
@@ -1,14 +1,27 @@
 sudo: false
+
 language: python
+
 before_install:
-  # for rpm-py-installer
```

Comment:
I guess you used "travis encrypt" to edit this file, and unfortunately by loading and writing out the YAML again it reformats the whole file and strips comments, like this one here.  It would be nice if that was avoided, e.g. using "git add -p" to keep only the change that you want.

Reply:
Thanks, I'll keep this in mind.